template/page-community.html.jinja

```diff
@@ -82,7 +73,174 @@
+
+      <div class="col d-lg-none d-block">
+        <div class="mt-10" id="networkNode-small"></div>
+      </div>
+
+      <div class="pt-2 pt-lg-5 card-grid card-grid-2xN text-cards">
+        <div class="text-card">
+          <a class="btn-arrow" href="/the-rippled-server.html">About the Server</a>
+          <p class="mt-3 mb-0">rippled is the core peer-to-peer server that manages the XRP Ledger.</p>
+        </div>
+    
+        <div class="text-card">
+          <a class="btn-arrow" href="/run-rippled-as-a-validator.html">Become a Validator </a>
+          <p class="mt-3 mb-0">Join the Unique Node List (UNL) to contribute to consensus on XRPL.</p>
+        </div>
+      
+        <div class="text-card">
+          <a class="btn-arrow" href="/install-rippled.html">Install & Configure</a>
+          <p class="mt-3 mb-0">Install, run, and update the rippled server.</p>
+        </div>
+      
+        <div class="text-card">
+          <a class="btn-arrow" href="/troubleshoot-the-rippled-server.html">Troubleshooting</a>
+          <p class="mt-3 mb-0">Troubleshoot any issues you’re having with the rippled server.</p>
+        </div>
+      </div>
+
+    </div>    
+  </div>
+</section>
+
+
+<!-- Grants -->
+<section class="container-new" id="xrpl-grants">  
+  <div class="card-grid card-grid-2xN">
+    <div class="col pr-2">
+      <div class="d-flex flex-column-reverse">
+        <h2 class="h4 h2-sm">{% trans %}Apply for funding for your next XRPL project{% endtrans %}</h2>
+        <h6 class="eyebrow mb-3">{% trans %}XRPL Grants{% endtrans %}</h6>
+      </div>
+      <p class="mb-lg-3 py-lg-4 pt-4 mb-0">{% trans %}The XRPL Grants program funds select open-source projects that contribute to the growing XRP Ledger community.{% endtrans %}</p>
+      
+      <div class="d-lg-block d-none">
+        <div class="mb-4 pb-3" id="xrplGrantsDark"></div>
+        <a class="btn btn-primary btn-arrow" target="_blank" href="https://xrplgrants.org/">{% trans %}Learn More{% endtrans %}</a> 
+      </div>
+    </div>
+
+    <div class="col">
+      <div class="mb-4 pb-3 mb-lg-3 pb-lg-5">
+        <h6 class="eyebrow mb-2">{% trans %}Awarded in a single grant{% endtrans %}</h6>
+        <img src="./assets/img/community/community-grants-1.svg" >
+      </div>
+      <div class="mb-4 pb-3 mb-lg-3 pb-lg-5">
+        <h6 class="eyebrow mb-2">{% trans %}Distributed to grant recipients{% endtrans %}</h6>
+        <img src="./assets/img/community/community-grants-2.svg" >
+      </div>
+      <div class="mb-4 pb-3 mb-lg-3 pb-lg-5">
+        <h6 class="eyebrow mb-2">{% trans %}Open-source projects funded {% endtrans %}</h6>
+        <img src="./assets/img/community/community-grants-3.svg" >
+      </div>
+
+      <div class="d-lg-none d-block mt-4 pt-3">
+        <div class="mb-4 pb-3" id="xrplGrantsDark-small"></div>
+        <a class="btn btn-primary btn-arrow" target="_blank" href="https://xrplgrants.org/">{% trans %}Learn More{% endtrans %}</a> 
```

Comment:
This link runs off the side of the screen in mobile. See screenshot:

![2022-06-08_131248_906891276](https://user-images.githubusercontent.com/7515597/172709899-a0f556ee-dbfb-4868-9e46-cd2945199d1f.png)